STYLE.md

```diff
@@ -17,6 +17,7 @@
   includes both const and non-const references.
 * Function names using camel case starting with a lower case letter (e.g., "doFoo()").
 * Struct/Class member variables have a '\_' postfix (e.g., "int foo\_;").
+* Enum values using all CAPS (e.g., "EXTERNAL_ONLY").
```

Comment:
hm, I'm not sure about this.
grep -ir 'enum class' gives the majority of the enum values in PascalCase.

Any particular reason we want to stick to the all CAPS?